### PR TITLE
gdalinfo: add page

### PR DESCRIPTION
**Pull request by JakobMiksch**
Add the CLI tool [gdalinfo](https://gdal.org/programs/gdalinfo.html) from the GDAL/OGR suite.

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
3.2.2

### CODE REVIEW

**Review by einverne — APPROVED** [review]
LGTM.

File: pages/common/gdalinfo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# gdalinfo
+
+> List various information about a GDAL supported raster dataset.
+> More information: <https://gdal.org/programs/gdalinfo.html>.
+
+- List supported raster formats:
```
Comment (EmilySeville7cfg) [inline pages/common/gdalinfo.md:6]:
```suggestion
- List all supported raster formats:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/gdalinfo.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`gdalinfo --formats`
```
[the remark this answers is not in the document]
Comment (JakobMiksch) [reply pages/common/gdalinfo.md:8]: It definitely exists. I tried it locally.
See the [official docs](https://gdal.org/programs/raster_common_options.html#cmdoption-formats). There `--formats` is listed as common option for all GDAL raster programs.

**Review by JakobMiksch — COMMENTED** [review]
@EmilySeville7cfg Thanks for your review. I addressed your notes.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by einverne — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Comment by JakobMiksch** [comment]
Thanks as well :-)

**Pull request merged**